packages/dev/core/src/Rendering/objectRenderer.ts

```diff
@@ -512,7 +512,7 @@ export class ObjectRenderer {
                 currentRenderList = defaultRenderList;
             }
 
-            if (!this._doNotChangeAspectRatio) {
+            if (!this.options.doNotChangeAspectRatio) {

```

Comment:
shouldnt the value be equal ? if not I wonder if this is the correct fix ?

Reply:
So should you set _doNotChangeAspectRatio  or remove it if unused as it sounds error prone ?

Reply:
Not even sure how I missed it... as this is literally on the same page ahah
![image](https://github.com/user-attachments/assets/0015c47c-6396-4c2b-a219-5ac3a8cd0954)